File: src/core/nativos/coleccion.js
export function ordenar(lista) {
    return [...lista].sort();
}

export function maximo(lista) {
    return Math.max(...lista);
}

export function minimo(lista) {
    return Math.min(...lista);
}

export function sin_duplicados(lista) {
    return Array.from(new Set(lista));
}

File: pCobra/corelibs/coleccion.py
"""Funciones para trabajar con colecciones."""


def ordenar(lista):
    """Devuelve una copia ordenada de *lista*."""
    return sorted(lista)


def maximo(lista):
    """Devuelve el valor máximo de *lista*."""
    return max(lista)


def minimo(lista):
    """Devuelve el valor mínimo de *lista*."""
    return min(lista)


def sin_duplicados(lista):
    """Retorna una lista sin elementos duplicados manteniendo el orden."""
    return list(dict.fromkeys(lista))
